### PR TITLE
chore: automagic contributor action

### LINKED ISSUE #34: [www] Update contributors list with github action
like here: https://github.com/typehero/typehero/pull/941

### PR DESCRIPTION
**Pull request by Hacksore**
Fix #34

if you wanna test this locally just create a `.env` file in the root of the monorepo with a `GITHUB_TOKEN`.

Then run `bun dl-contrib` and it should run the script to refresh the contributors.

### CODE REVIEW

File: .github/workflows/contributors.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Install dependencies
+        run: bun install
+      - name: Run contributors script
+        run: bun dl-contrib
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
+        with:
+          token: ${{ secrets.GH_TOKEN }}
```
Comment (Hacksore) [inline .github/workflows/contributors.yaml:51]: @danstepanov you will need to generate a [PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens) and add it to the secrets called `GH_TOKEN` that has access to create pull requests.

**Review by danstepanov — APPROVED** [review]
LGTM

**Comment by danstepanov** [comment]
@Hacksore This is great and confirmed to be working! How might we automatically update the contributors in the README?

**Comment by Hacksore** [comment]
@danstepanov created #220 to track this effort, it's close but needs a tiny bit more love.

**Pull request merged**